.github/workflows/test.yaml

```diff
@@ -0,0 +1,34 @@
+name: test
```

Comment:
Don't know if you are planning to add back [a pass/fail badge to the readme](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge), but if so I'd advocate for tweaking the name on the workflow. It might be possible for us to control the display name for this elsewhere but I've not found it in a quick read through that docs page.